docs/management/manage-index-patterns.asciidoc

```diff
@@ -7,60 +7,64 @@ To customize the data fields in your index pattern, you can add runtime fields t
 [[runtime-fields]]
 === Explore your data with runtime fields 
 
-Runtime fields are fields that you add to documents after you've ingested, and are evaluated at query time. With runtime fields, you allow for a smaller index and faster ingest time so that you can use less resources and reduce your operating costs. You can use runtime fields anywhere index patterns are used.
+Runtime fields are fields that you add to documents after you've ingested your data, and are evaluated at query time. With runtime fields, you allow for a smaller index and faster ingest time so that you can use less resources and reduce your operating costs. You can use runtime fields anywhere index patterns are used.
 
-When you use runtime fields, you can:
+With runtime fields, you can:
 
-* Define fields for a specific use without modifying the underlying schema. 
+* Define fields for a specific use case without modifying the underlying schema. 
 
 * Override the returned values from index fields.
 
-* Start working on your data without first understanding the structure. 
+* Start working on your data without understanding the structure. 
 
 * Add fields to existing documents without reindexing your data. 
 
 * Explore runtime field data in *Discover*.
 
-* Create visualizations with runtime field data using *Lens*, *Maps*, and *TSVB*.
+* Create visualizations with runtime fields to display on a dashboard.
 
 WARNING: Runtime fields can impact {kib} performance. When you run a query, {es} uses the fields you index first to shorten the response time. 
 Index the fields that you commonly search for and filter on, such as `timestamp`, then use runtime fields to limit the number of fields {es} uses to calculate values. 
 
-For more information, refer to {ref}/runtime.html[Runtime fields].
+For detailed information on how to use runtime fields with {es}, refer to {ref}/runtime.html[Runtime fields].
 
 [float]
 [[create-runtime-fields]]
-==== Create runtime fields
+==== Add runtime fields
```

Comment:
Good catch, which made me realize there's an inconsistency in the UI.  The menu item is "Add field" but the form is "Create field".